core/Names.cc

```diff
@@ -361,6 +361,12 @@ NameRef NameRef::addEq(GlobalState &gs) const {
     return gs.enterNameUTF8(nameEq);
 }
 
+NameRef NameRef::lookupWithEq(const GlobalState &gs) const {
+    auto name = this->dataUtf8(gs);
+    string nameEq = absl::StrCat(name->utf8, "=");
+    return gs.lookupNameUTF8(nameEq);
+}
+
```

Comment:
looks dead? is this used somewhere?

Reply:
https://github.com/sorbet/sorbet/blob/9878d4597ba1b9d37a1e96b2383ec926ace170e5/packager/rbi_gen.cc#L611

Reply:
ah, github had collapsed the diff in that file. thanks!